### DISCUSSION TITLE
Expo Router - Grabbing query params from url string

### DISCUSSION THREAD

**Discussion by doloboyz** [expo/router#507] [Q&A]
Hey y'all, 
I had a question regarding how I should go about grabbing query params from a url string after clicking on an email hyperlink. 

For example, let's say I have the following string: 

`http://localhost:19000/account/verify?token=1_Fld4-Vsf8iQgbcak2Rh2zr-k5AdWlGpua71D4dGSDK7`

I'd like to grab the token ( `1_Fld4-Vsf8iQgbcak2Rh2zr-k5AdWlGpua71D4dGSDK7` )  so I can post it to my verification API. How would I go about grabbing this token once the user clicks the hyperlink.

Thanks for any help!

**Comment by likeSo** [comment]
See https://expo.github.io/router/docs/migration/react-navigation/params
